sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@
 - Removed `Blob` from the name of lease options and return types, like `AcquireBlobLeaseOptions` was renamed to `AcquireLeaseOptions`.
 - Removed `Blob` prefix from the name of structs `BlobGeoReplication`, `BlobLeaseStatus`, `BlobLeaseState`, `BlobLeaseDurationType`, `BlobAnalyticsLogging`, `BlobMetrics`, `BlobCorsRule`, `BlobRetentionPolicy`, `BlobStaticWebsite`, `BlobArchiveStatus` and `BlobGeoReplictionStatus`.
 - Removed `RequestId` from return types.
+- Changed `BodyStream` parameter in upload functions from pointer to reference.
```

Comment:
Would we consider append/stage block as "upload"? We may want to include the other APIs too in the CL.